tests/run/cpdef_extern_func.pyx

```diff
@@ -1,5 +1,6 @@
 # cython: c_string_type=str
 # cython: c_string_encoding=ascii
+# distutils: extra_compile_args=-fpermissive
```

Comment:
I doubt that this is a C compiler option that is globally available in all supported C/C++ compilers. I'd rather change the test to use a different C function (but one with two parameters to make use of keyword arguments).